SpatialGDK/Source/SpatialGDK/Private/Utils/ComponentReader.cpp

```diff
@@ -308,7 +308,7 @@ void ComponentReader::ApplyProperty(Schema_Object* Object, Schema_FieldId FieldI
 					// it's part of a streaming level that hasn't been streamed in. Native Unreal networking sets reference to nullptr and continues.
 					// So we do the same.
 					FString FullPath;
-					improbable::GetFullPathFromUnrealObjectReference(ObjectRef, FullPath);
+					SpatialGDK::GetFullPathFromUnrealObjectReference(ObjectRef, FullPath);
```

Comment:
```suggestion
					GetFullPathFromUnrealObjectReference(ObjectRef, FullPath);
```
Unnecessary since we are in the namespace.

Reply:
Looks like your PR addresses this.